pyproject.toml

```diff
@@ -45,17 +45,17 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 
-lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", tag = "v0.3.0" }
-
-torch = "1.8.1"
-torchvision = "0.9.1"
-h5py = "^3.1.0"
-ninja = "^1.10.2"
-matplotlib = "^3.5.1"
-numpy = "^1.22.2"
-scipy = "^1.8.0"
-pillow = "^9.0.1"
-pytest = "^7.0.1"
+lava-nc = { git = "https://github.com/lava-nc/lava.git", branch = "main", tag = "v0.4.0" }
```

Comment:
Can we change it to point to the latest version of lava? There are some fixes I am issuing that are needed to fix the broken changes in lava 0.4.0

Reply:
ACK